config/locales/zh.yml

```diff
@@ -1492,6 +1492,7 @@ time.formats.event_timestamp: '%B %-d, %Y at %-l:%M %p'
 time.formats.event_timestamp_js: '%{year} %{month} %{day}, %{hour}:%{minute} %{day_period}'
 time.formats.event_timestamp_utc: '%B %-d, %Y， %-l:%M %p UTC'
 time.formats.event_timestamp_with_zone: '%B %-d, %Y，%-l:%M %p %Z'
+time.formats.full_date: '%Y 年 %B %-d 日'
```

Comment:
I'm slightly worried about adding a new field for this one use case. Should we check to see if the `time.formats.event_date` field shouldn't be changed to match this format?

Reply:
There also needs to be a space after the month digit and the following "月"

Reply:
this i can't seem to change, `%B` comes with "月". If i change the format to us `%b` and then add " 月" I get translation missing. If i change the date format to instead use `%m` and add " 月" what is rendered is `05 月`. I think we need design to weigh in on what they would prefer b/c we can't render it the way it is in the current design.

Reply:
`%b` is used for short name months, so in English it would be like "Jan", "Feb", "Mar" instead of "January", "February", "March", etc. When I tried using it, I got:
```
"Translation missing: zh.date.abbr_month_names"
```
which just means we would need to add an array of `date.abbr_month_names` just like the `date.month_names` that we already have


The `月` comes from `%B` for full month names, which loads from our own translation files:
https://github.com/18F/identity-idp/blob/37afb7efb468486efe3f7ed50ad7600f6329fc37/config/locales/zh.yml#L421-L430

Reply:
I don't know how accurate this is, but when I use google translate, it seems that 05 月 = May which I believe is correct? I'm also personally okay with us trying our best here and sending it to DOS to weigh in post-merging.

Reply:
`月` means month, so `5月` is "the fifth month" = May

Reply:
@zachmargolis  the full months that we have in our translation file don't have the space between the number and character, which is wha the design was calling for. 
@rutvigupta-design  Sounds like perhaps `05 月` will be ok instead of what the design originally called for which was`5 月`.

Reply:
Since `05 月` means the same thing as `5 月` (I think), if we're finding that it's not possible to render `5 月`, I'd say that's fine. When we submit this to DOS, we can double-check that this isn't hugely different from the original rendering.

Reply:
i'm going to change it to use `05`. Now that Zack as confirmed that we are using the above code that was linked to render the month I would say we can put a space between the number and word BUT it would change every occurrence across the app.

Reply:
@svalexander I know the translations have a space between the number and word, but I wonder if it's okay to keep them together if we didn't want to change every occurrence across the app? Would it be a significant lift to change this later on, if DOS doesn't agree?

Reply:
Ok we can go with `5月` as it currently and see what DOS says. If they want the space for sure we can update to use the `05 月` version. That would be a small change.